GNUmakefile

```diff
@@ -3,7 +3,7 @@ GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 PKG_NAME=datadog
 DIR=~/.terraform.d/plugins
 ZORKIAN_VERSION=master
-API_CLIENT_VERSION=master
```

Comment:
We don't generally update this to contain the new release tag. The assumption is that when `make update-go-client` is run, you want to *update*, so fixing `API_CLIENT_VERSION` to the tag is not really useful.

Reply:
Yeah that was unwanted.